tests/stories/LegacyTemplate.stories.svelte

```diff
@@ -0,0 +1,43 @@
+<script context="module" lang="ts">
+  import { Story, Template } from '@storybook/addon-svelte-csf';
+  import type { Meta } from '@storybook/svelte';
+
+  import LegacyTemplate from './LegacyTemplate.svelte';
+
+  /**
+   * Description set explicitly in the comment above export const meta.
+   *
+   * Multiline supported. And also Markdown syntax:
+   *
+   * * **Bold**,
+   * * _Italic_,
+   * * `Code`.
+   */
+  export const meta = {
+    title: 'LegacyTemplate',
+    component: LegacyTemplate,
+    tags: ['autodocs'],
+  } satisfies Meta<LegacyTemplate>;
+</script>
+
+<script lang="ts">
+  let count = $state(0);
+
+  function handleClick() {
+    count += 1;
+  }
+</script>
+
+<Template let:args let:context>
+  <p>Using default template</p>
+  <LegacyTemplate {...args} onclick={handleClick} />
+</Template>
+
+<Template id="rounded" let:args>
+  <p>Using rounded template</p>
+  <LegacyTemplate {...args} onclick={handleClick} rounded />
+</Template>
+
+<Story name="Default" />
+
+<Story name="Rounded" template="rounded" />
```

Comment:
I think _this_ is causing the Chromatic failure. I expect you'll see the same if you build the Storybook locally and serve it, maybe even in dev mode too.

This is the raw story as opened in an iframe: https://6042cb92fd1bb200234586ee-poqxheiipz.chromatic.com/iframe.html?id=tests-legacytemplate--default&viewMode=story

First error:

![image](https://github.com/user-attachments/assets/9a2bf5b1-f229-4aed-be4f-318e0c187536)

Leading to this:

![image](https://github.com/user-attachments/assets/1bac99d2-38ab-49cc-9240-172f5eb16765)

I think this is the mismatch in snippet reference that I've commented on earlier. The snippet will be named `rounded()`, but after the codemod the Story will reference it as `children={Rounded}` - and `Rounded` is coincidentally the export name of the story, so it's in fact not referencing the snippet but the story defined as `const Rounded = stories[...]` after post-transformation.

Reply:
Haha, I shot myself in the foot with this one. I was troubled with how the hell, then I remember:

https://github.com/storybookjs/addon-svelte-csf/blob/89bd7f1375c391f76aec40390e66e3c0b73fbae2/src/compiler/pre-transform/codemods/legacy-story.ts#L231-L238


Thanks for this amazing work, and I'm going to make a note to myself. Ensure that runtime works by actually manually verifying the runtime 😅